vite-plugin/playground/src/components/regular-script/1-Basic.vue

```diff
@@ -1,19 +1,17 @@
 <template>
+  <h3>Basic</h3>
+
   <div>
     <q-btn label="Click me" />
+
     <q-icon name="map" />
```

Comment:
First of all, it's something that's got carried over from the old example folder.

Just speculating: bundling a CSS file is probably faster than processing a much bigger JS file, detecting and dropping the unused ones.

I tested it and it doesn't make more than ~0.5s difference (_total run time ~12.5s_). One way is faster than the other sometimes. So, overall, now worth it.

As neither the CSS nor the JS version is not very special, they should not be worthy of testing from the Vite plugin perspective. So, I am leaving the code as is since it's shorter and simpler.